test_check_limits.py

```diff
@@ -0,0 +1,17 @@
+from check_limits import Check_Limits
+
+test_obj = Check_Limits()
+
+if __name__ == '__main__':
+    assert(test_obj.battery_is_ok([{25: True}, {70: True}, {0.76: True}], language='German') is True)
```

Comment:
The warning needs to be displayed if the value is in tolerance range. Whereas here it is provided as an option to the user. Is that intentional?

Reply:
Yes @prithamvshetty Sir,

If we look at the documentation, it says:

> Keep in mind: Though we are starting with warning levels for all parameters, customers may give feedback to have warnings only for some parameters and not others. Minimize the change needed for such 'tuning'.

So this code should work in future if user doesn't want warning for any of the feature, and it should be as simple as {parameter value : warning=False} and True for other parameters.

Reply:
Clear , thanks..👍